examples/window/window_settings.rs

```diff
@@ -63,3 +75,17 @@ fn cycle_cursor_icon(
         window.set_cursor_icon(ICONS[*index]);
     }
 }
+
+/// This system changes the window icon every second.
+fn change_window_icon(mut windows: ResMut<Windows>, time: Res<Time>) {
```

Comment:
I don't like this; it's very artificial (which will mean that future contributors will do strange things trying not to break it). I can't think of a case where I've ever seen a window icon change at runtime.

Reply:
Discord changes their icon at runtime for notification messages.

Also that concept of icons changing at runtime feels like you could make a fascinating one time experience playing off expectations of windowing.

Reply:
Would it be better to entirely remove the runtime icon changing?

Reply:
I opted to remove this as it's out of the scope of the original issue and I couldn't think of a remotely common use-case.